packages/curve-ui-kit/src/shared/ui/stories/Balance.stories.tsx

```diff
@@ -0,0 +1,111 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { Balance } from '../Balance'
+
+const meta: Meta<typeof Balance> = {
+  title: 'UI Kit/Widgets/Balance',
+  component: Balance,
+  argTypes: {
+    symbol: {
+      control: 'text',
+      description: 'The token symbol',
+    },
+    balance: {
+      control: 'number',
+      description: 'The token balance',
+    },
+    notionalValue: {
+      control: 'number',
+      description: 'The USD value of the balance',
+    },
+    max: {
+      control: 'select',
+      options: ['balance', 'button', 'off'],
+      description: 'The max button mode',
+    },
+    hideIcon: {
+      control: 'boolean',
+      description: 'Whether to hide the wallet icon',
+    },
+    onMax: {
+      action: 'onMax',
+      description: 'Callback when max button is clicked',
+    },
+  },
+  args: {
+    symbol: 'ETH',
+    balance: 1.234,
+    max: 'off',
+    hideIcon: false,
+    onMax: (maxValue: number) => {
+      // eslint-disable-next-line no-console
+      console.log('Max value:', maxValue)
```

Comment:
We can use console.log for testing and console.info for stuff that stays in the codebase
https://github.com/curvefi/curve-frontend/blob/532951c78308ab27606976d095a1437e97a6f10f/packages/eslint-config-custom/index.js#L31

Btw, [storybook has actions](https://storybook.js.org/docs/essentials/actions) for this purpose

```suggestion
      console.info('Max value:', maxValue)
```

Reply:
Oops, forgot to fix this. Will do it in the large input PR